### ISSUE TITLE
Device Manger reports "Invalid Configuration Descriptor"

### ISSUE THREAD

**Issue by Lillythefox** [polhenarejos/pico-openpgp#15]
Hello, I believe the Pico Patcher is not working, the device is showing up as "Invalid Configuration Descriptor" and none of my software can detect it (GPG, kleopatra and opensc). As you see below in the event viewer logs it mentions "00FF2000" the default descriptor.

![image](https://github.com/polhenarejos/pico-openpgp/assets/30854494/f897f619-b7ef-4924-8018-eef7acba1f20)

```
Device USB\VID_0000&PID_0006\5&2d531919&0&1 was configured.

Driver Name: usb.inf
Class Guid: {36fc9e60-c465-11cf-8056-444553540000}
Driver Date: 06/21/2006
Driver Version: 10.0.19041.4355
Driver Provider: Microsoft
Driver Section: BADDEVICE.Dev.NT
Driver Rank: 0xFF0000
Matching Device Id: USB\CONFIGURATION_DESCRIPTOR_VALIDATION_FAILURE
Outranked Drivers: usb.inf:USB\CONFIGURATION_DESCRIPTOR_VALIDATION_FAILURE:**00FF2000**
Device Updated: false
Parent Device: USB\ROOT_HUB30\4&30961835&0&0
```

**Comment by imkuang** [comment]
Same problem with version 2.0, version 1.12 works fine.

**Comment by polhenarejos** [comment]
Fixed in 11cb855

**Comment by ItalianJack** [comment]
This appears to still be an issue with release 2.2. Compiling while setting the IDs manually works fine, however the online tool doesn't seem to change them from default.
